pkg/asset/manifests/ingress.go

```diff
@@ -119,13 +119,15 @@ func (ing *Ingress) generateClusterConfig(config *types.InstallConfig) ([]byte,
 
 	switch config.Platform.Name() {
 	case aws.Name:
-		obj.Spec.LoadBalancer = configv1.LoadBalancer{
-			Platform: configv1.IngressPlatformSpec{
-				AWS: &configv1.AWSIngressSpec{
-					Type: config.AWS.LBType,
+		if len(config.AWS.LBType) > 0 {
+			obj.Spec.LoadBalancer = configv1.LoadBalancer{
+				Platform: configv1.IngressPlatformSpec{
+					AWS: &configv1.AWSIngressSpec{
+						Type: config.AWS.LBType,
+					},
+					Type: configv1.AWSPlatformType,
```

Comment:
Is there concern with leaving `ingress.spec.LoadBalancer.Platform.Type` as empty `""` if the install `config.AWS.LBType` is not configured?

Reply:
the const type defined is either Classic or NLB. So "" shall not be allowed to add in the ingressconfig for the LBType.

Reply:
The ingress operator should handle this case without issue, but we can set an explicit default to "Classic" if that makes more sense to do.

Reply:
But you can't leave it as "" as only two values are allowed i.e Classic and NLB. I don't think setting Classic in Ingress.Spec.Loadbalancer  is needed as it is optional.

Reply:
Question about handling the LBType not being set was originally raised here: https://github.com/openshift/installer/pull/6074/commits/fbd2ea07afc59239e66ae0eff26cd776a5c8cd41#r971049072.

What happens when the manifest is created with LBType value is set to "Classic" or "NLB" initially on day 0 and the customer updates just the LBType field to "" day 2 keeping everything else in the manifest the same? Does the ingress controller be updated to take care of this?